packages/crayons-core/src/utils/index.ts

```diff
@@ -259,3 +259,23 @@ export const cyclicDecrement = (value: number, maxValue: number): number => {
   value--;
   return value < 0 ? maxValue : value;
 };
+
+export const createGuid = () => {
```

Comment:
lets use an external lib to handle generation of uuid
https://www.npmjs.com/package/uuid